### ISSUE TITLE
Elastic Search mapper_parsing_exception

### ISSUE THREAD

**Issue by jcteague** [simianhacker/bunyan-elasticsearch#14]
I'm getting the following error, using the documentation on the readme.

```
Elasticsearch Stream Error: Error: [mapper_parsing_exception] failed to find type parsed [multi_field] for [name]
```
I followed the instructions regarding posting the template.json file as well.

I'm targeting the AWS ES service v 5.1

**Comment by spotshare-nick** [comment]
I'm also having this issue.  Maybe we can get a slightly updated template file?

**Comment by spotshare-nick** [comment]
http://stackoverflow.com/questions/40301061/elasticsearch-5-mapperparserexception-with-multi-field -- depreciated in ES 5

**Comment by jcteague** [comment]
did that SO answer work for you?

**Comment by spotshare-nick** [comment]
No, StackOverflow wasn't enough to get it fixed.  I did find a proper template file though, and that worked.  https://github.com/logstash-plugins/logstash-output-elasticsearch/tree/master/lib/logstash/outputs/elasticsearch

Version 2.x
Version 5.x
Version 6.x

**Comment by jcteague** [comment]
cool, I'll give that a shot.  Thanks
